### ISSUE TITLE
Add include_fs -option

### ISSUE THREAD

**Issue by hannut** [grke/burp#337]
Hello!

Please add include_fs -option or equivalent, so it would be easier choose what to backup. For example I have hundreds of machines and would like to backup everything on local filesystems ext2/3/4, btrfs, ufs, zfs, etc on all machines. Now I can give a very long list of all the fs'es I don't wan t to backup, but no the other way around. The different os/versions make the list of excluded filesystems rather long and error prone.

This is something I have in mind:

include_fs=ext4
include_fs=btrfs
include_fs=ufs
...

include=/
exclude = /.snapshots/
exclude = /tmp/
...

-hannut

**Comment by grke** [comment]
Fair enough. It will be some time before I get to this, because there are many other things that I think are higher priority.
